org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/extensions/contentmodel/model/CMDocument.java

```diff
@@ -32,4 +32,8 @@
 	 */
 	CMElementDeclaration findCMElement(DOMElement element, String namespace);
 
+	String getURI();
+
+	String setURI(String uri);

```

Comment:
IMHO this setUri should not exists and uri should be initialized in the constructor of each CMDocument implementation.

Reply:
Alright, will update that.